UefiTestingPkg/FunctionalSystemTests/MpManagement/App/MpManagementTestApp.inf

```diff
@@ -0,0 +1,44 @@
+## @file
+# UEFI shell unit test application for MP management driver.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010006
+  BASE_NAME                      = MpManagementTestApp
+  FILE_GUID                      = B6ED4248-1EEE-4E70-BCB2-E167E42B1230
+  MODULE_TYPE                    = UEFI_APPLICATION
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = MpManagementTestApp
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32 X64 AARCH64
```

Comment:
If this is currently only for ARM, shouldn't IA32 X64 be removed?

Reply:
The test application is general as it only depends on the `MP_MANAGEMENT_PROTOCOL`. But the implementation for `MP_MANAGEMENT_PROTOCOL` is only for ARM right now. I will remove the X86 marks for the protocol DXE driver. But I think this app's `VALID_ARCHITECTURES` could stay as it is. Please let me know if you think otherwise.

Reply:
That should be fine